src/customize-pop/application-settings.md

```diff
@@ -0,0 +1,63 @@
+| Usage | Display an application's disk usage. Click on the `Storage` button to view space used by the application itself, data created by the application, and temporary data cached by the application. Click `Clear Cache` to clear an application's locally stored data.
+
+## Notifications
+
+Settings to control application notification behavior are located in the `Settings` ➞ `Notifications` tab.
+
+![Notifications Settings](/images/application-settings/notifications-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Do Not Disturb | Disable all application notifications globally. |
+| Lock Screen Notifications | Enable setting toggling for the `Lock Screen Notifications` and `Show Message Content on Lock Screen` options. |
+
+Application notifications appear in the upper center of the screen. Click the date and time to view recent notifications.
+
+![Recent Notifications](/images/application-settings/recent-notifications.png)
+
+### Configure Notification Behavior for a Single Application
+
+Select an application from the list to configure its notification behavior.
+
+![Single App Options](/images/application-settings/single-app-options.png)
+
+| Option | Function |
+|--------|----------|
+| Notifications | Toggle an application's notifications on or off. |
+| Sound Alerts | Toggle sound alerts that will accompany notifications for this application. |
+| Notification Popups | Enable notification popups on the desktop; notifications will continue to appear in the notification list. |
+| Show Message Content in Popups | Include the notification's written content in the popup. |
+| Lock Screen Notifications | Display application notifications at the login screen when the computer is locked. These settings cannot be enabled unless the Lock Screen Notifications setting is enabled. |
+| Show Message Content on Lock Screen | Displays an application's written notification content in the login screen when the computer is locked. |
+
+## Default Applications
+
+Options to choose default applications for Web browsing, Email, Calendars, Music, Videos, and Photos are located in `Settings` ➞ `Default Applications`.
+
+![Default Applications](/images/application-settings/default-applications.png)
+
+Designate an application as the default by selecting it from the dropdown menu. If no appropriate application is installed to handle a media type, the dropdown selection will be grayed out. <!--link to installing applications section when complete-->
+
+![Select Default](/images/application-settings/select-default.png)
+
+> **Default Applications vs. Default Handlers**:
+> Multiple applications can be designated as default handlers for file types. Use the `Default Applications` menu to specify which application should **always** launch when handling a specific media type, even if other appropriate applications are installed on the system.
```

Comment:
I don't understand the distinction you're trying to make here. The default application option will set the default handler for a large number of file types at once (e.g. the Photos default application will set the default handler for `.jpg`, `.png`, `.webp`, etc.) If you wanted to explain how to set default handlers more granularity, that would need to be elaborated on. I don't understand the emphasis on default being what `**always**` launches since that is what default would mean (unless you right-click and select a different application manually, in which case "always" isn't accurate.)

Reply:
I'll remove this and expand on the Default Handlers section.

Reply:
This section has been split out into sub-chapters for my next commit, and I've expanded on setting default apps/file handlers. I'm going to resolve this comment, but feel free to unresolve if you feel necessary.